yupana-utils/src/main/java/org/yupana/utils/KazakhStemmerTokenFilter.java

```diff
@@ -0,0 +1,91 @@
+package org.yupana.utils;
+
+import org.apache.lucene.analysis.TokenFilter;
+import org.apache.lucene.analysis.TokenStream;
+import org.apache.lucene.analysis.tokenattributes.CharTermAttribute;
+import org.apache.lucene.analysis.tokenattributes.KeywordAttribute;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class KazakhStemmerTokenFilter extends TokenFilter {
```

Comment:
Do we really need to inherite `TokenFilter` here? I suppose this might be something similar to `RussianLightStemmer`.